src/main/java/phonebook/PhoneDirectory.java

```diff
@@ -0,0 +1,42 @@
+package phonebook;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class PhoneDirectory {
+    private List<Record> records = new ArrayList<Record>();
+
+    public PhoneDirectory(List<Record> records) {
+        this.records = records;
+    }
+
+    public List<Record> getRecords() {
```

Comment:
```suggestion
    protected List<Record> getRecords() {
```
якщо лише для тестів, то protected краще